### PR TITLE
Half Life sound effects, HEV Suit and scientist changes

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: Added the technobabble emote, which makes a special sound if you're wearing the scientist labcoat
add: Added a tie to the scientists starting outfit. You will wear it.
add: Added a different scientist starting uniform. Sprited by Fantastic Fwoosh and Mr Epic Sax. Thank you kings.
add: Changed the scientist labcoat to an older one that looks better.
add: The HEV Suit, sprited by Mr. Epic Sax. Thank you king.
tweak: Replaced the researcher's hardsuit with the HEV Suit.
soundadd: Changed soundeffect for stepping on normal tiles or metal ones with the classic HL1 one. Grass, sand and wood are unchanged.
soundadd: Added sounds to most buttons, also from HL1
soundadd: Added two other sounds for triggering the grenade, from the HECU grunts
soundadd: Changed the voting start sound to G-Man telling you to choose
soundadd: Changed the forge Hammer sounds.
soundadd: Added a sound for light switches.
sounddel: removed catwalk1-5.ogg because I was unable to find where they were used in code
/:cl:

## About The Pull Request
I reckon this needs a testmerge first, they might be too loud or too quiet.

## Why It's Good For The Game
FORGET ABOUT FREEMAN

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
hmm yes code

**Comment by karmaisblackandbluepilled** [comment]
if you ask me what the fuck I am doing I will shoot you @Neo-0

**Comment by Neo-0** [comment]
very cool

**Comment by karmaisblackandbluepilled** [comment]
what the hell?
I don't get those errors on my end

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -447,21 +447,12 @@
...
-	var/explosion_detection_dist = 21
-	scan_reagents = TRUE
-	actions_types = list(/datum/action/item_action/toggle_helmet_light, /datum/action/item_action/toggle_research_scanner)
+	//RD suit moved to hippie folder
 
-/obj/item/clothing/head/helmet/space/hardsuit/rd/Initialize()
+/obj/item/clothing/suit/space/hardsuit/rd/equipped(mob/user, slot)
 	. = ..()
+	if(!ishuman(user))
+
```
Comment (Neotwr) [inline code/modules/clothing/spacesuits/hardsuit.dm]: what's it? there's nothing under

Comment (karmaisblackandbluepilled) [reply code/modules/clothing/spacesuits/hardsuit.dm:455]: fuck

Comment (karmaisblackandbluepilled) [reply code/modules/clothing/spacesuits/hardsuit.dm]: right fixed

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -483,21 +470,9 @@
...
-	item_state = "hardsuit-rd"
-	resistance_flags = ACID_PROOF | FIRE_PROOF
-	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT //Same as an emergency firesuit. Not ideal for extended exposure.
-	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/gun/energy/wormhole_projector,
-	/obj/item/hand_tele, /obj/item/aicard)
-	armor = list("melee" = 30, "bullet" = 5, "laser" = 10, "energy" = 5, "bomb" = 100, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 80)
-	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/rd
-
-
+	if(slot == SLOT_WEAR_SUIT)
```
Comment (Neotwr) [inline code/modules/clothing/spacesuits/hardsuit.dm]: ITEM_SLOT_OCLOTHING

File: hippiestation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 3]
```diff
@@ -34,6 +34,69 @@
...
+	armor = list("melee" = 30, "bullet" = 10, "laser" = 10, "energy" = 5, "bomb" = 100, "bio" = 100, "rad" = 60, "fire" = 60, "acid" = 80)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/rd
+	var/heal_threshold = 10 //yeah n-word, i'm stealing medbot code, what of it
+	var/injection_amount = 10
+
+/obj/item/clothing/suit/space/hardsuit/rd/equipped(mob/user, slot)
+	. = ..()
+	if(!ishuman(user))
+		return
+	if(slot == SLOT_WEAR_SUIT)
```
Comment (Neotwr) [inline hippiestation/code/modules/clothing/spacesuits/hardsuit.dm]: ITEM_SLOT_OCLOTHING

File: hippiestation/code/modules/mob/living/emote.dm [hunk 4]
```diff
@@ -1,6 +1,33 @@
...
+	key_third_person = "technobabbles"
+	message = "says something scientific."
+	message_mime = "pretends to be a researcher."
+	only_forced_audio = FALSE
+	vary = TRUE
+
+/datum/emote/living/technobabble/get_sound(mob/living/user)
+	if(ishuman(user))
+		user.adjustOxyLoss(user.scream_oxyloss)
+		if(is_type_in_list(user.get_item_by_slot(SLOT_WEAR_SUIT), list(/obj/item/clothing/suit/toggle/labcoat/science, /obj/item/clothing/suit/toggle/labcoat/chemist, /obj/item/clothing/suit/space/hardsuit/rd, /obj/item/clothing/suit/toggle/labcoat/virologist, /obj/item/clothing/suit/toggle/labcoat/genetics, /obj/item/clothing/suit/toggle/labcoat/cmo)))
```
Comment (Neotwr) [inline hippiestation/code/modules/mob/living/emote.dm]: ITEM_SLOT_OCLOTHING

File: hippiestation/code/modules/clothing/spacesuits/hardsuit.dm [hunk 5]
```diff
@@ -34,6 +34,69 @@
...
+	scan_reagents = TRUE
+	actions_types = list(/datum/action/item_action/toggle_research_scanner)
+
+/obj/item/clothing/head/helmet/space/hardsuit/rd/Initialize()
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_EXPLOSION, .proc/sense_explosion)
+
+/obj/item/clothing/head/helmet/space/hardsuit/rd/equipped(mob/living/carbon/human/user, slot)
+	..()
+	if (slot == SLOT_HEAD)
```
Comment (Neotwr) [inline hippiestation/code/modules/clothing/spacesuits/hardsuit.dm]: ITEM_SLOT_HEAD

**Comment by karmaisblackandbluepilled** [comment]
i don't understand how cloning the pr from the other repo fucked so much shit, what the hell

**Comment by karmaisblackandbluepilled** [comment]
alright I'm gonna test a few things to see if they're working and if it's working I'll push again

**Comment by karmaisblackandbluepilled** [comment]
works? Perhaps?

**Comment by karmaisblackandbluepilled** [comment]
love how RegisterSignal(SSdcs, COMSIG_GLOB_EXPLOSION, .proc/sense_explosion) isn't in my game but it is here

**Comment by karmaisblackandbluepilled** [comment]
seems to work time to shill it

**Comment by mkikt4743** [comment]
c'mon c'mon

**Comment by karmaisblackandbluepilled** [comment]
nice

**Comment by mkikt4743** [comment]
MERGE NOW!!!

**Comment by Neo-0** [comment]
since I think this already has enough votes
also @karmaisblackandbluepilled close https://github.com/HippieStation/HippieStation-Fallout-Shelter/pull/3

**Comment by KingTerrytheTerrible** [comment]
these sound effects are awful, what the fuck? we're not hl2rp

**Comment by KingTerrytheTerrible** [comment]
revert this shit

**Pull request merged**